Core/scripts/dirac-install.py

```diff
@@ -72,6 +72,12 @@
 
    It will install DIRAC v6r20-pre16
 
+   You can install an extension of diracos.
+
+   for example::
+
+     dirac-install -r v9r4-pre2 -l LHCb --dirac-os --dirac-os-version=LHCb:master
```

Comment:
I guess that the example should read (no equals sign):

  dirac-install -r v9r4-pre2 -l LHCb --dirac-os --dirac-os-version LHCb:master